CHANGES.rst

```diff
@@ -112,12 +112,15 @@ astropy.uncertainty
 
 astropy.units
 ^^^^^^^^^^^^^
+
 - ``Quantity.to`` has gained a ``copy`` option to allow copies to be avoided
   when the units do not change. [#10517]
 
 - Added the ``spat`` unit of solid angle that represents the full sphere.
   [#10726]
 
+- The ``bar`` unit is no longer wrongly considered an SI unit. [#10586]
```

Comment:
Does this belong under API Changes instead?

Reply:
Hmm, yes, you're right - will put it under bug-fix (with a `[ci skip]`, so current commit pass is what will count). I think API change is not quite right since it doesn't change how one accesses the unit.

Reply:
Actually, let me put it in API changes after all - it is not something we really want to back-port, and it does change some decomposition results.